cmd/create_automated_pr.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/hacbs-contract/ec-cli/internal/ecGit"
+)
+
+func createAutomatedPR() *cobra.Command {
+	var data = struct {
+		componentRepoURL  string
+		destinationBranch string
+		prBranchName      string
+		prTitle           string
+		prBody            string
+		diffFilePath      string
+	}{
+		destinationBranch: "main",
+	}
+	cmd := &cobra.Command{
+		Use:   "create-automated-pr",
```

Comment:
Is it automated if I need to run it? I'd simplify:
```suggestion
		Use:   "create-pr",
```

When we have another provider in addition to GitHub, there might be a need to separate those, let's worry about that then...